working-together/README.md

```diff
@@ -41,6 +41,19 @@ and valued during remote meetings that happen over video conferencing software.
   - Consider calling in if you have an unreliable internet connection.
   - Be understanding of interruptions that may happen with remote work,
     e.g. children, pets, other people who might be sharing space with your teammate.
+- **Cultural Diversity:** In a diverse workplace, people will have diverse cultures, accents and forms of speech, and this is totally fine. Always try to make communication as clear as possible.
+  - **Avoid** whenever possible:
+    - Using slang when talking about an important subject
+    - Using passive voice (i.e. *"You will be asked some questions"*)
```

Comment:
I'm unsure why this is important. Could you clarify?

Reply:
This is a recommendation from the [AAAL (American Association for Applied Linguistics)](https://www.aaal.org/guidelines-for-communication-rights##)!

Using passive voice can confuse non-native speakers as the wording can seem unusual depending on their mother language!

If we consider the example phrase, it would sound something like:
"You will be ask some questions", which could lead to a potential misunderstanding of "You will be asking some questions".

Active voice is something present in every language and normally what we are taught first in English, as well!

Reply:
That makes sense. Thanks for the clarification! I wonder if we could include that example in the text. It was very helpful!

Reply:
I like that idea! Gonna try to add it :)